composer.json

```diff
@@ -56,6 +57,13 @@
     {
       "type": "path",
       "url": "packages/wp-phpunit-polyfill"
+    },
+    {
+      "type": "path",
+      "url": "packages/apiclient-services-subscribewithgoogle",
+      "options": {
+        "symlink": false
```

Comment:
This option causes the package to be mirrored/copied into the `vendor` directory instead of using a symlink which is the default. This was necessary to avoid strange breaking behavior with our dependency prefixing.